### ISSUE TITLE
count_fixed! macro doesn't seem to expand properly

### ISSUE THREAD

**Issue by andrew-d** [Geal/nom#53]
Hi there,

I tried fixing this myself, but I don't understand why it's not working - sorry :disappointed: 

Essentially, this works (in a `chain!`):

```rust
        e_res2:     count_fixed!( call!(le_u16), u16, 10 ) ~
```

But this does not, despite [this macro case](https://github.com/Geal/nom/blob/3cb3e34a2f3812a0c9229e50055746dcd7e9373b/src/macros.rs#L1537-1539):

```rust
        e_res2:     count_fixed!( le_u16, u16, 10 ) ~
```

[Here's](https://github.com/andrew-d/binscope-rs/blob/3c688d6a13014d8ce05b178a9c71a14d9b67f7ff/pefile/src/parsers.rs#L31) where I'm using it, if that's helpful.

**Comment by Geal** [comment]
This happens because there was an ambiguity in `count_fixed` pattern matching. I fixed it in b47043c2 but it requires a syntax change. I'll publish a new crate soon

**Comment by Geal** [comment]
Hi! Is it working alright for you?
